src/sql/azdata.d.ts

```diff
@@ -3575,6 +3577,11 @@ declare module 'azdata' {
 		 * Minimum height for editor component
 		 */
 		minimumHeight: number;
+
+		/**
+		 * Title of editor
+		 */
+		title: string;
```

Comment:
Where is this coming from? Should go in proposed first by default

Reply:
whoops yeah, looks like it was being used without ever being in the API